tmpl.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1-alpine as builder
```

Comment:
Can we add a comment to clarify what this file is for?

As far as I can tell, `seihon` reads it as template, when no CLI flag is given (`--template=./tmpl.Dockerfile`). This should be made somewhat explicit - either with a comment here, or an explicit call to `seihon` in the `.travis.yml`

Reply:
I added a comment in the `tmpl.Dockerfile`.